modules/container-service/managed-cluster/main.bicep

```diff
@@ -676,7 +676,7 @@ resource managedCluster_roleAssignments 'Microsoft.Authorization/roleAssignments
   scope: managedCluster
 }]
 
-resource dnsZone 'Microsoft.Network/dnsZones@2018-05-01' existing = if (dnsZoneResourceId != null && webApplicationRoutingEnabled) {
+resource dnsZone 'Microsoft.Network/dnsZones@2018-05-01' existing = if (enableDnsZoneContributorRoleAssignment == true && dnsZoneResourceId != null && webApplicationRoutingEnabled) {
```

Comment:
Hey @coolhome,
sorry I only just got arround. Thank you for your contribution.
Could you please run the `utilities/tools/Set-Module.ps1`  script on the module? This will update the `main.json` & readme file